src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -349,7 +349,7 @@ static void F(IEnumerable<int[]> x)
             comp.VerifyDiagnostics();
         }
 
-        [Fact]
+        [ConditionalFact(typeof(NoIOperationValidation), Reason = "Timeouts")]
```

Comment:
>Timeouts [](start = 67, length = 8)

Should we have an issue to track disable tests like that?

Reply:
I don't think that's particularly useful, as I'm not currently thinking that we'll be attempting to renable them in the future.

---
In reply to: [529820501](https://github.com/dotnet/roslyn/pull/49534#discussion_r529820501) [](ancestors = 529820501)